documentation/source/contributors.rst

```diff
@@ -6,7 +6,13 @@ Contributors
    McGregor
    Grimwood
 
-cocotb was developed by `Potential Ventures <https://potential.ventures>`_ with the support of
+cocotb is developed and maintained by an active community.
+Our GitHub repository contains a list of all `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_.
+
+Since the beginning of 2019 and the 1.1 release,
+the `Free and Open Source Silicon Foundation <https://www.fossi-foundation.org>`_
+provides the umbrella organisation.
```

Comment:
Please reword to something along those lines:

> Since mid-2018 cocotb is an independent project under the umbrella of the Free and Open Source Silicon Foundation. The FOSSi Foundation provides the cocotb project with financial, legal and administrative support, and holds cocotb assets.


(The 1.1 release in January 2019 is the first release which came out of that transition, but it took a while to get there.)

Reply:
Done.